index.bs

```diff
@@ -177,6 +177,9 @@ time, calling methods SHOULD have the same behaviour, etc. However, the user
 agent MAY transition out of Picture-in-Picture when the video element enters a
 state that is considered not compatible with Picture-in-Picture.
 
+Styles applied to |video| (such as ''display: none'') don't apply in the
```

Comment:
I would recommend using wording such as "MUST NOT apply" instead of "don't apply". Also, I would recommend keeping the example more vague such as "(such as opacity, visibility, transforms, ...)".